@navikt/core/css/modal.css

```diff
@@ -1,69 +1,114 @@
-.ReactModal__Body--open {
-  overflow: hidden;
-}
-
 .navds-modal {
   background-color: var(--ac-modal-bg, var(--a-surface-default));
-  border-radius: var(--a-border-radius-medium);
-  display: block;
-  position: relative;
-  overflow: auto;
-  max-height: 100%;
+  border: 1px solid var(--ac-modal-border, var(--a-border-subtle));
+  border-radius: var(--a-border-radius-large);
   box-shadow: var(--a-shadow-xlarge);
+  padding: 0;
+  position: fixed;
 }
 
-.navds-modal__content {
-  padding: var(--a-spacing-4);
+.navds-modal[open] {
+  display: flex;
+  flex-direction: column;
 }
 
-.navds-modal:focus-visible,
-.navds-modal--focus {
-  box-shadow: var(--a-shadow-focus);
-  outline: none;
+.navds-modal--polyfilled {
+  top: 50%;
+  transform: translate(0, -50%);
+  overflow: auto;
+  max-height: 80%;
```

Comment:
Bør `max-height` ha en `minmax`?
```css
max-height: minmax(80%, Xrem)
```

Reply:
Forstår ikke helt hva du mener her. Ser ut som `minmax` bare kan brukes med grid.

Reply:
Ah my bad, tror man kan bruke bare `max` her istedenfor
```
max-height: max(80%, Xrem)
```

Da for å hindre at modalen blir kjempelang hvis man har en høy skjerm. Men vet ikke om dette er et problem i det hele tatt så kan se bort fra kommentaren hvis det ikke trengs

Reply:
Modalen blir uansett ikke lenger enn innholdet (unntatt kanskje på mobil som nevnt under), og hvis du har en høy skjerm så tenker jeg det bare er en fordel å utnytte hele høyden.